Sources/SwiftInspectorVisitors/AssociatedtypeVisitor.swift

```diff
@@ -52,7 +52,7 @@ public final class AssociatedtypeVisitor: SyntaxVisitor {
   }
 }
 
-public struct AssociatedtypeInfo: Codable, Equatable {
+public struct AssociatedtypeInfo: Codable, Hashable {
```

Comment:
this change fixes a build break on `main` that I introduced when my last PR merged without first rebasing on top of latest `main`.